doc/releases/release-notes-4.0.rst

```diff
@@ -549,6 +549,11 @@ Drivers and Sensors
 
 * MIPI-DBI
 
+  * Added bitbang MIPI-DBI driver, supporting 8080 and 6800 mode
+    (:dtcompatible:`zephyr,mipi-dbi-bitbang`).
+  * Added support STM32 FMC memory controller (:dtcompatible:`st,stm32-fmc-mipi-dbi`).
```

Comment:
```suggestion
  * Added support for STM32 FMC memory controller (:dtcompatible:`st,stm32-fmc-mipi-dbi`).
```

Reply:
Fixed (unlike my ability to proofread, apparently)- thanks for catching this